src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -339,11 +379,47 @@ const HealthCardDisplay = () => {
                       {patientData.immunizations.map((item) => (
                         <div key={item.fullUrl}>
                           <ListItem>
-                            <HealthCardVaccination immunization={item.resource} />
+                            <HealthCardVaccination
+                              immunization={item.resource}
+                            />
                           </ListItem>
                         </div>
                       ))}
                     </List>
+                    <Grid
+                      container
+                      xs={10.5}
+                      justifyContent="left"
+                      alignItems="flex-end"
+                    >
+                      <Box
```

Comment:
This looks much better but the styling is a bit off. It looks like there is an inherited max-width that throws it off, if you override that it looks much better.